CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEfficiencyHarvester.cc

```diff
@@ -509,12 +503,12 @@ void SiStripHitEfficiencyHarvester::printTotalStatistics(
   int subdetfound[5];
```

Comment:
can we just set these equal to {0} here and remove lines 506-510?

Reply:
Well, maybe,  but what's the benefit?

Reply:
It's a bit cleaner.

Reply:
Since I had to touch the module again, I addressed this at a50bccd41f329742bbc6ec7a59ff2c160406e005